### ISSUE TITLE
Response body examples include description and value fields

### ISSUE THREAD

**Issue by simondean** [Mermade/widdershins#102]
Hi.  The OpenAPI 3.0.x spec seems to suggest example response bodies can be specified like this: 

```yaml
            responses:
                '200': 
                    description: Responds with a greeting
                    content: 
                        application/json:
                            schema: 
                                $ref: schemas/get-greeting/response/200.json
                            examples:
                                default:
                                    description: Successfull response
                                    value:
                                        $ref: examples/get-greeting/response/200/default.json
```

When I do this with widdershins + shins, the right hand side JSON example in the Slate HTML looks like this: 

```
Example responses:

{
  "description": "Successfull response",
  "value": {
    "greeting": "Hello!"
  }
}
```

It looks like widdershins might be using the `examples.default` object rather than the `examples.default.value` object for the example it outputs?  

I've created an example codebase here: https://github.com/simondean/widdershins-issues/blob/master/response-examples

**Comment by simondean** [comment]
Hi.  Here's the current HTML output: https://simondean.github.io/widdershins-issues/response-examples/#getgreeting 

The actual output looks like this: 

```
Example responses:

{
  "description": "Successfull response",
  "value": {
    "greeting": "Hello!"
  }
}
```

The expected output would be this: 

```
Example responses:

{
  "greeting": "Hello!"
}
```

**Comment by MikeRalphson** [comment]
I'm not sure embedding a `$ref` in the `value` property of an [example object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#example-object) is valid OAS 3 (the `$ref` should be at the level of your `default` key, and to an `example object` in `#/components/examples` or an external document) , but that's separate to your issue.

I've made a fix to pick up the example object from the `.value` property of the `example object`. This area could probably be improved, to show descriptions and multiple examples, but for now, this should be an improvement. Thanks for pointing it out.

**Comment by simondean** [comment]
@MikeRalphson that's an interesting observation about using `ref$` in a `value` property.  Hadn't thought of that.  

I tried using `externalValue` but I don't think `widdershins` shows the renders the contents of `externalValue`.  I was also put off by this part of the description of `externalValue` in the spec: `This provides the capability to reference examples that cannot easily be included in JSON or YAML documents.` - as my JSON schema is easily included in JSON 😄 

It'd would be nice if OpenAPI supporting using `$ref` in the `value` property...

I've just tried the latest version of widdershins and the fix is working well.  Thanks
